Userland/Libraries/LibJS/Bytecode/Operand.h

```diff
@@ -14,13 +14,11 @@ namespace JS::Bytecode {
 class Operand {
 public:
     enum class Type {
-        Invalid,
```

Comment:
Changes in this file in this commit seem like they should be squashed into the "add Operand" commit